src/python/pants/scm/change_calculator.py

```diff
@@ -1,166 +0,0 @@
-# coding=utf-8
-# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
-                        unicode_literals, with_statement)
-
```

Comment:
Since this file is gone, you should delete the target that refers to it from `src/python/pants/scm/BUILD`... afterward, you should run `./pants list ::` to confirm that all references to `src/python/pants/scm:change_calculator` are deleted.